docs/learning/SupportedPlatforms.md

```diff
@@ -16,11 +16,8 @@ They should run on most Windows, Linux and MacOS versions with [Tier 1 support](
 | - | - |
 | Node 14.x | ✔️ |

```

Comment:
Is there something specific that we need in >14.15? I may just update the description to mention the LTS versions are the only ones supported, that way it applies to all major versions rather than having to specify each.

Reply:
What do you think of the update?

Reply:
I'd still prefer to specify in engines - that would prevent anyone from using 13.

I didn't have anything specific in mind, but it's worth noting that (among other things) this would give us the flexibility to use [NAPI 7](https://nodejs.org/api/n-api.html#n_api_node_api_version_matrix) in the native addon.  Without setting a minimum 14.x version, we're stuck with NAPI 6 for all of our 3.x generation.

Also, FWIW, if we say the minimum is 12.**22**, and 14.17 we can even use NAPI 8.  There's not much added in either 7 or 8, but since changing min node version is considered breaking, if we don't set this now and decide we want to use anything in either sometime down the road, we'd be stuck waiting for iTwin.js 4.0...

Reply:
I'd rather say that we support the LTS version of the 12, 14 and 16 (when it comes out) instead of being more specific to just the NAPI 8 since going to LTS should get us v7...

I can agree with updating the engines setting. I'll do that now.

Reply:
@wgoehrig convinced me that it makes sense to update only a couple more minor releases of Node in order for us to future proof a bit by being able to use N-API 8. Just updated the new minimums.